cmd/kuberhealthy/util.go

```diff
@@ -54,9 +54,11 @@ func notifyChanLimiter(maxSpeed time.Duration, inChan chan struct{}, outChan cha
 			select {
 			case <-time.After(maxSpeed):
 				outChan <- struct{}{}
+				break
 			case <-inChan:
 				log.Println("channel notify limiter witnessed an upstream message on inChan and is waiting an additional", maxSpeed, "before sending output")
 			}
+			return
```

Comment:
this was old / temp fix! i believe you can now pull / merge with what's in master!